### PR TITLE
MODLD-334: IDENTIFIER type added to ID mappers

### PR DESCRIPTION
**Pull request by PBobylev**

### CODE REVIEW

File: src/main/java/org/folio/linked/data/model/entity/Resource.java [hunk 1]
```diff
@@ -137,8 +138,10 @@ public Resource addType(@NonNull ResourceTypeEntity type) {
     return this;
   }
 
-  public Resource addType(@NonNull org.folio.ld.dictionary.ResourceTypeDictionary typeDictionary) {
-    this.addType(new ResourceTypeEntity(typeDictionary.getHash(), typeDictionary.getUri(), null));
+  public Resource addType(@NonNull org.folio.ld.dictionary.ResourceTypeDictionary... typeDictionary) {
```
Comment (aleksei-pronichev) [inline src/main/java/org/folio/linked/data/model/entity/Resource.java:141]: `addTypes`

**Review by aleksei-pronichev — APPROVED** [review]

**Review by askhat-abishev — APPROVED** [review]

File: src/main/java/org/folio/linked/data/model/entity/Resource.java (hunk 1 above)
Comment (PBobylev) [reply src/main/java/org/folio/linked/data/model/entity/Resource.java:141]: thx, fixed

File: src/main/java/org/folio/linked/data/model/entity/Resource.java [hunk 2]
```diff
@@ -132,13 +133,15 @@ public Set<ResourceTypeEntity> getTypes() {
     return types;
   }
 
-  public Resource addType(@NonNull ResourceTypeEntity type) {
+  public Resource addTypes(@NonNull ResourceTypeEntity type) {
```
Comment (pkjacob) [inline src/main/java/org/folio/linked/data/model/entity/Resource.java:136]: minor - it was better to leave the name of this method as `addType` right? (as this method accept only one object as input)

Comment (PBobylev) [reply src/main/java/org/folio/linked/data/model/entity/Resource.java:136]: oh yep, it was refactored by my IDE automatically, let me fix it

**Pull request merged**